code/src/java/pcgen/gui2/dialog/TipOfTheDay.java

```diff
@@ -132,10 +135,9 @@ private void initUI()
 		FontManipulation.xxlarge(lblDidYouKnow);
 		lblDidYouKnow.setOpaque(true);
 
-		tipText = new JLabelPane();
 		tipText.setBorder(null);
 		tipText.setFocusable(false);
-		tipText.addHyperlinkListener(new Hyperactive());
+//		tipText.addHyperlinkListener(new Hyperactive());
```

Comment:
We plan on using this? Otherwise commented out code is a best practice no-no and just accrues code debt.

Reply:
This is actually more of a TODO. Agreed that commented code is bad, but in this case its not unreasonable.